apps/admin/frontend/src/screens/hardware_diagnostics_screen.tsx

```diff
@@ -1,13 +1,24 @@
-import { H2, Icons, P, PrinterStatusDisplay } from '@votingworks/ui';
+import { H2, H6, Icons, P, PrinterStatusDisplay } from '@votingworks/ui';
 
 import { format } from '@votingworks/utils';
 import type { DiagnosticsRecord } from '@votingworks/admin-backend';
 import type { BatteryInfo } from '@votingworks/backend';
 import { NavigationScreen } from '../components/navigation_screen';
-import { getDiagnosticRecords, getPrinterStatus, systemCallApi } from '../api';
+import {
+  getDiagnosticRecords,
+  getPrinterStatus,
+  getApplicationDiskSpaceSummary,
+  systemCallApi,
+} from '../api';
 import { Loading } from '../components/loading';
 import { PrintTestPageButton } from '../components/print_diagnostic_button';
 
+function roundToGigabytes(kilobytes: number): number {
+  return Math.round(kilobytes / 100_000) / 10;
+}
+
+const DISK_SPACE_WARN_LEVEL = 0.05;
```

Comment:
nit: measurement unit would be useful here for readability (`DISK_SPACE_RATIO_WARN_LEVEL` maybe?)